.ci/run.sh

```diff
@@ -10,4 +10,8 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     pyenv activate conan
 fi
 
+## create Environment Variables
```

Comment:
this is not necessary, environment variable is automatically set by bincrafters-package-tools